creator-node/src/apiHelpers.js

```diff
@@ -1,4 +1,10 @@
+const config = require('./config')
+const Web3 = require('web3')
+// TODO: use web3 provider url ?
+const web3 = new Web3(config.get('ethProviderUrl'))
```

Comment:
@vicky-g i think you can actually just initialize it with no provider. i assume we're not using the existing web3 object because it's attached to the express server and we can't require it?
```
const web3 = new Web3()
```

Reply:
yeah, the web3 object is in the req.app object. it was simpler to just require the direct web3 object and not have to add another param to the successResposne to pass in the web3 obj